charts/incubator/warrior/values.yaml

```
@@ -0,0 +1,35 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: atdr.meo.ws/archiveteam/warrior-dockerfile
+  # -- image tag
+  tag: latest
+  # -- image pull policy
+  pullPolicy: IfNotPresent
```

When there is only a `latest` image available it might be better to set this to `Always` by default.

```suggestion
  pullPolicy: Always
```